content/blog/2022/2022-12-26-lc0-release-v0290.md

```diff
@@ -0,0 +1,55 @@
+---
+title: "Lc0 release v0.29.0"
+# slug: "blog-post-title"
+published: 2022-12-26
+#draft: true
+author: "borg"
+# image: "name"
+# cover: "cover.png" # Default
+---
+
+After a full year without a Lc0 release, [v0.29.0](https://github.com/LeelaChessZero/lc0/releases/tag/v0.29.0) was finally released a few weeks ago, so this is a perfect opportunity to make a blog post about it and explain why it took so long. Moreover, this way we'll get at least one blog post in 2022...
```

Comment:
As I mentioned in Discord, the overal tone is a bit more apologetic (that we are so slow) rather than explaining.  

It's totally fine to keep it as it is, but if you would like to do modifications, [this](https://discord.com/channels/425419482568196106/439388231146930176/1057055922557550752) chatgpt text is actually a good inspiration.